modules/service/src/main/scala/gem/Service.scala

```diff
@@ -67,23 +77,25 @@ final class Service[M[_]: Sync: LiftIO] private (private val xa: Transactor[M],
 
     import gem.ocs2.{ Importer, OdbClient }
 
-    /** Constructs a program that fetches the corresponding observation from the
-      * indicated ODB and then stores it in the database, replacing any
-      * existing observation with the same id.
-      */
+    /**
+     * Constructs a program that fetches the corresponding observation from the
+     * indicated ODB and then stores it in the database, replacing any
+     * existing observation with the same id.
+     */
     def importObservation(host: String, oid: Observation.Id): M[Either[String, Unit]] =
       log.log(user, s"ocs2.importObservation($host, $oid)") {
         OdbClient.fetchObservation[IO](host, oid).liftIO[M].flatMap { _.traverse { case (o, ds) =>
```

Comment:
Seems like there should be a combinator for `.flatMap(_.traverse` but there's not.